conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from brainzutils.musicbrainz_db import init_db_engine
+
+
+@pytest.fixture(scope="session")
+def engine():
+    init_db_engine("postgresql://musicbrainz@musicbrainz_db/musicbrainz_db")
```

Comment:
this shouldn't be hard-coded.
There are some pytest plugins that allow for transactional tests, e.g. https://github.com/jeancochrane/pytest-flask-sqlalchemy, however I'm not sure that it's necessary, as we're only using the database in read-only mode.